modules/cssn/src/Plugin/Block/PersonaBlock.php

```diff
@@ -78,11 +79,15 @@ public function build() {
       $last_name = $user_entity->get('field_user_last_name')->value;
       $institution = $user_entity->get('field_institution')->value;
       $roles = $user_entity->getRoles();
+      $is_student = array_search('student', $roles) !== FALSE;
+      $academic_status = $is_student
+        ? $user_entity->get('field_academic_status')->value : '';
+      $academic_status = $this->getAcademicStatuslongform($academic_status);
```

Comment:
@a-pasquale @protitude -- the `field_academic_status` is made of short keys & long values.  i don't know how to get the long values out of the field.  if there were a way to do that, then the above translating function could go away

Reply:
Resolved by next commit.